portality/lib/dataobj.py

```diff
@@ -272,7 +272,7 @@ class DataObj(object):
         "license": string_canonicalise(["CC BY", "CC BY-NC", "CC BY-NC-ND", "CC BY-NC-SA", "CC BY-ND", "CC BY-SA", "Not CC-like"], allow_fail=True),
         "persistent_identifier_scheme": string_canonicalise(["None", "DOI", "Handles", "ARK"], allow_fail=True),
         "format": string_canonicalise(["PDF", "HTML", "ePUB", "XML"], allow_fail=True),
-        "deposit_policy": string_canonicalise(["None", "Sherpa/Romeo", "Dulcinea", "OAKlist", "Diadorim"], allow_fail=True),
+        "deposit_policy": string_canonicalise(["None", "Sherpa/Romeo", "Dulcinea", "OAKlist", "Diadorim", "Mir@bel"], allow_fail=True),
```

Comment:
Unrelated to the additionl of Mir@bel, what is this `OAKlist`, maybe is a legacy value, I think it can be removed (either now or in future)